docs/docsite/rst/porting_guides/porting_guide_base_2.11.rst

```diff
@@ -21,7 +21,7 @@ The complete list of porting guides can be found at :ref:`porting guides <portin
 Playbook
 ========
 
-No notable changes
+* The ``jinja2_native`` setting now does not affect the template module which implicitly returns strings. The template lookup and the rest of the Jinja2 expressions still operate based on the ``jinja2_native`` setting.
```

Comment:
What's the reason for not switching the `template` lookup?  My initial thought was that is should follow the same behavior as the module.  Otherwise you cannot easily template out a JSON document via the lookup.

Reply:
My thought was that lookups can be used "inline" in Jinja expressions which might be potentially useful to have native types there but I don't have a specific example.

This seems confusing though. Should we perhaps have a toggle in the template lookup (`False` by default) for native jinja?

And since I don't have a specific example for where leaving native types for the template lookup would be useful, I am ok with it behaving the same as the module.